### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by floe**
rearranged the readme a bit, not sure if the build instructions are sufficient?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,78 @@
...
+
+## Building
+
+Install dependencies (`sudo apt install libopencv-dev build-essential v4l2loopback-dkms curl`).
+
+Clone this repository with `git clone --recursive https://github.com/floe/backscrub.git`.
+To speed up the checkout you can additionally pass `--depth=1` to `git clone`.
+This is okay, if you only want to download and build the code, however, for development it is not recommended.
+
+Use `cmake` to build the project: create a subfolder (e.g. `build`), change to that folder and run: `cmake && make -j4` (please replace "4" with the number of your cores).
```
Comment (phlash) [inline README.md:23]: I usually work in the source folder with:
```bash
% cmake -B build
% cmake --build build
```
but that's personal pref/taste. If using the `cd` approach, then the parent directory needs to be referenced ie:
`cmake .. && make -j <n>`

File: README.md [hunk 2]
```diff
@@ -6,6 +6,78 @@
...
+
+Alternatively, you can also run `make` to build everything (should also clone and build Tensorflow Lite).
```
Comment (phlash) [inline README.md:25]: We may want to deprecate this now, it doesn't produce the same results (no XNNPACK), and Tensorflow have all but dropped support.

File: README.md [hunk 3]
```diff
@@ -6,6 +6,78 @@
...
+
+## Usage
+
+First, load the v4l2loopback module (extra settings needed to make Chrome work):
+```
+sudo modprobe v4l2loopback devices=1 max_buffers=2 exclusive_caps=1 card_label="VirtualCam"
```
Comment (phlash) [inline README.md:31]: I like to force the device numbering here, avoids confusion:
```bash
sudo modprobe v4l2loopback video_nr=10 label="Virtual Cam" max_buffers=2 exclusive_caps=1
```

File: README.md [hunk 4]
```diff
@@ -6,6 +6,78 @@
...
+    - Linux kernel 5.6 (stock package)
+    - OpenCV 4.2.0 (stock package)
+    - V4L2-Loopback 0.12.5 (stock package)
+    - Tensorflow Lite 2.5.0 (from [repo](https://github.com/tensorflow/tensorflow/tree/v2.5.0/tensorflow/lite))
+  - Ubuntu 18.04.5, x86-64
+    - Linux kernel 4.15 (stock package)
+    - OpenCV 3.2.0 (stock package)
+    - V4L2-Loopback 0.10.0 (stock package)
+    - Tensorflow Lite 2.1.0 (from [repo](https://github.com/tensorflow/tensorflow/tree/v2.1.0/tensorflow/lite))
+
```
Comment (phlash) [inline README.md:54]: Can also add my Debian 10 (stable) with stock kernel, opencv, v4l2loopback and TFlite 2.5.0

**Review by phlash — COMMENTED** [review]
Might want to point folks at the Issues / milestones lists for what's coming up.

File: README.md [hunk 5]
```diff
@@ -6,6 +6,78 @@
 ![Screenshots with my stupid grinning face](images/screenshot.jpg)
 (Credits for the nice backgrounds to [Mary Sabell](https://dribbble.com/shots/4686178-Bauhaus-Poster) and [PhotoFunia](https://photofunia.com/effects/retro-wave))
 
+## Maintainers
+
+  * Phil Ashby (@phlash)
+  * Benny Baumann (@benbe)
```
Comment (BenBE) [inline README.md:12]: Preferred capitalization of the nickname is as follows … But all lower is acceptable too.
```suggestion
  * Benny Baumann (@BenBE)
```

File: README.md (hunk 1 above)
Comment (BenBE) [reply README.md:23]: ACK on the `cd` approach.

File: README.md (hunk 2 above)
Comment (BenBE) [reply README.md:25]:
```suggestion
**Deprecated**: Another option to build everything is to run `make` in the root directory of the repository. While this will download and build all dependencies, it comes with a few drawbacks like missing support for XNNPACK. Also this might break with newer versions of Tensorflow Lite as upstream support for this option has been removed. Use at you own risk.
```

File: README.md (hunk 3 above)
Comment (BenBE) [reply README.md:31]: Maybe also document the configuration via `/etc/modprobe.d/v4l_loopback.conf` by setting up a configuration like:
```
# V4L loopback driver
options v4l2loopback max_buffers=2
options v4l2loopback exclusive_caps=1
# options v4l2loopback devices=4
options v4l2loopback card_label="VirtualCam 0,VirtualCam 1,VirtualCam 2,VirtualCam 3"
```

Doing so reduces the complexity to load the driver. Also adding `/etc/modules-load.d/v4l2loopback.conf` with the following content allows to auto-load the driver on every boot:
```
v4l2loopback
```

File: README.md [hunk 6]
```diff
@@ -6,6 +6,78 @@
...
+First, load the v4l2loopback module (extra settings needed to make Chrome work):
+```
+sudo modprobe v4l2loopback devices=1 max_buffers=2 exclusive_caps=1 card_label="VirtualCam"
+```
+Then, run deepseg (-d -d for full debug, -c for capture device, -v for virtual device):
+```
+./deepseg -d -d -c /dev/video0 -v /dev/video1
+```
+
+Some cameras like a *Logitec Brio* might need to switch the video source to `MJPG` by passing `-f MJPG` in order to allow for higher resolutions.
```
Comment (BenBE) [inline README.md:38]:
```suggestion
Some cameras (like e.g. `Logitec Brio`) need to switch the video source to `MJPG` by passing `-f MJPG` in order for higher resolutions to become available for use.
```

**Review by BenBE — CHANGES_REQUESTED** [review]
Also maybe reference the license of this project directly in the readme. Just recently had some user for one of my projects who was somewhat surprised, when I rejected one contribution, because of license mismatch (contribution of Apache licensed code in AGPLv3 project of mine). Thus making the license really clear with as little complications as possible, should mitigate this situation somewhat.

File: README.md (hunk 4 above)
Comment (floe) [reply README.md:67]: Please do :-)

**Comment by floe** [comment]
BTW: once the readme is updated, should we tag a release v0.3 or so?

**Review by BenBE — APPROVED** [review]

**Comment by floe** [comment]
Closes #82 .
